doc/source/index.rst

```diff
@@ -318,8 +318,9 @@ Papers
    :maxdepth: -1
    :caption: Ray RLlib
 
-   rllib.rst
+   rllib/index.rst
```

Comment:
Will start moving all rllib docs files into the `rllib` sub-folder to the rst's will no longer be in the top-leve directory. Analogous to how tune docs do this.

Reply:
good idea!